README.md

```diff
@@ -56,6 +41,7 @@ The Stream has multiple configuration groups:
 ### Stream Options
 
 There are two configs within the Stream Options group.
+[Missing Content]
```

Comment:
This mentions two configs but only lists source objects. If source objects it the only config, then remove the two configs text.